components/inputs/input-date.js

```diff
@@ -155,25 +170,44 @@ class InputDate extends LocalizeStaticMixin(LitElement) {
 		this._dropdown = this.shadowRoot.querySelector('d2l-dropdown-content');
 		this._calendar = this.shadowRoot.querySelector('d2l-calendar');
 
+		this.addEventListener('blur', this._handleBlur);
 		this.addEventListener('d2l-localize-behavior-language-changed', () => {
 			this._dateTimeDescriptor = getDateTimeDescriptorShared(true);
-			this.requestUpdate();
+			this.requestUpdate().then(() => {
+				const width = Math.ceil(parseFloat(getComputedStyle(this.shadowRoot.querySelector('.d2l-input-date-hidden-content')).getPropertyValue('width')));
+				this._hiddenContentWidth = `${width}px`;
+			});
 		});
+
+		this._formattedValue = this.emptyText ? this.emptyText : '';
+
+		await (document.fonts ? document.fonts.ready : Promise.resolve());
+		const width = Math.ceil(parseFloat(getComputedStyle(this.shadowRoot.querySelector('.d2l-input-date-hidden-content')).getPropertyValue('width')));
+		this._hiddenContentWidth = `${width}px`;
 	}
 
 	render() {
+		const shortDateFormat = (this._dateTimeDescriptor.formats.dateFormats.short).toUpperCase();
+		const inputTextWidth = `calc(${this._hiddenContentWidth} + 0.75rem + 2px)`; // text and icon width + paddingRight + border width
 		return html`
+			<div class="d2l-input-date-hidden-content">
+				<div><d2l-icon icon="tier1:calendar"></d2l-icon>${formatISODateInUserCalDescriptor('2020-12-20')}</div>
+				<div><d2l-icon icon="tier1:calendar"></d2l-icon>${shortDateFormat}</div>
+				<div><d2l-icon icon="tier1:calendar"></d2l-icon>${this.emptyText}</div>
```

Comment:
Good idea! The `visibility: hidden` for sure hides this from screen readers?

Reply:
It does seem to (tested on NVDA and VoiceOver, and googling seems to indicate that it should). I'll add `aria-hidden="true"` too just in case.